test/middleware-test.js

```diff
@@ -573,6 +573,18 @@ describe('input-validation middleware tests', function () {
                     done();
                 });
         });
+        it('valid request - should pass validation', function (done) {
```

Comment:
please add another test that sends `Capital-Letters` header as `capital-letters` - it should pass of course it's just to nail the behavior